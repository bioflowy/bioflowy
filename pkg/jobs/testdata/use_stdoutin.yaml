children:
  - args:
      - "echo"
      - "this stdoutin test"
    stdout:
      pipe: "pipe"
  - args:
      - "sh"
      - "-c"
      - "nl > use_stdoutin.yaml.result"
    stdin:
      pipe: "pipe"
