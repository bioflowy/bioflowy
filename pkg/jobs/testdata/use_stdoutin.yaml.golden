     1	this stdoutin test
